### DISCUSSION TITLE
Didn't Receive Code when I want login

### DISCUSSION THREAD

**Discussion by mohammadprgrm20123** [codemagic-ci-cd/codemagic-docs#2608] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

No matter how hard I try, the code is not sent to me

### Expected results

''''''''

### Actual results

''''''

**Comment by dam-ease** [comment]
Hello @mohammadprgrm20123,
Have you also checked your spam/junk folders as well?
You can also try using a different browser. Please try to clean the cache from your browser as well.

**Reply by mohammadprgrm20123** [reply]
I receive the code but I receive it late

**Reply by dam-ease** [reply]
Thanks for your response!
So, all good now right?

**Reply by mohammadprgrm20123** [reply]
hi
 I tried several times but I can't log in

please follow up

**Reply by dam-ease** [reply]
Hello,
Sorry about that!
Please share your email id with me so I can check this out.

**Reply by dam-ease** [reply]
Thanks for your response.
Checking through the time interval from the logs, OTPs were sent without delays. The reason you didn't receive might be because of internet connection or please check if you enabled greylisting which temporarily rejects incoming emails.
You can also try authenticating by using Git providers OAuth.

**Reply by dam-ease** [reply]
Hello,
Any update on this?
If there is anything else I can assist you with, please don't hesitate to let me know.

**Reply by IbrahimKhanLS** [reply]
@dam-ease Hey, I am trying to signup using "****@***.com" email. I am not receiving any code on my group email.

**Reply by dam-ease** [reply, marked as answer]
Hello @IbrahimKhanLS.
Please can you try the sign up again now?
For security reasons, I removed your email address from the thread.

**Reply by lordkkjmix** [reply]
Hello i don't receive the code too when i try to login, my email is **

**Reply by dam-ease** [reply]
Hello @lordkkjmix 
Checking through the logs, I see the authentication code has been sent, can you confirm this please?

**Reply by IbrahimKhanLS** [reply]
@dam-ease yes it worked, thanks
